src/VisualStudio/Core/Def/Implementation/Preview/PreviewUpdater.PreviewDialogWorkspace.cs

```diff
@@ -4,14 +4,14 @@
 using System.Threading.Tasks;
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.Editor.Shared.Preview;
-using Microsoft.CodeAnalysis.Shared.Extensions;
 using Microsoft.CodeAnalysis.Text;
 
 namespace Microsoft.VisualStudio.LanguageServices.Implementation.Preview
 {
     internal partial class PreviewUpdater
     {
-        private class PreviewDialogWorkspace : PreviewWorkspace
+        // internal for testing
```

Comment:
If this is the standard approach that is fine by me.  At a glance it looked like a one off comment which is why I made my suggestion.